components/prism-java.js

```diff
@@ -55,7 +55,13 @@
 		'operator': {
 			pattern: /(^|[^.])(?:<<=?|>>>?=?|->|--|\+\+|&&|\|\||::|[?:~]|[-+*/%&|^!=<>]=?)/m,
 			lookbehind: true
-		}
+		},
+		'constant': /\b[A-Z][A-Z_\d]+\b/,
+		'variable': [
+			{
+				pattern: /\b(?:args|log(?:ger)?|value)\b/
```

Comment:
Why did you add this again?

Reply:
I think `args`, `log`, `logger` and `value` are common, so this pattern is added.

Reply:
I don't think this is sufficient justification to add arbitrary words. Please remove this.

Reply:
done